### DISCUSSION TITLE
percy long notes or visually shortened long notes for mania not showing up in lazer

### DISCUSSION THREAD

**Discussion by MrZee12** [ppy/osu#14516] [Q&A] [answered]
When i imported my stable skin my long notes are not showing up entirely. For some people this helps or is necessary to them for reading including myself because i can't read normal long notes and this will be my turn off and some for getting into lazer if this isn't fixed.

Comparison videos: 

Osu! Stable (this is what it should look like)
https://user-images.githubusercontent.com/89574459/130928654-87aa7d30-03d5-4fd3-a0a1-6f1bd10ad50f.mp4

Osu! Lazer (the problem)
https://user-images.githubusercontent.com/89574459/130928750-abd92824-baf7-4050-906b-3f5d69cb6036.mp4

Skin Link: https://www.mediafire.com/file/viqjoinqnajfpyu/-_%2523InugamiKorone__%2528SSS_Beta_Mr.Zee_Edit_v3%2529_%25284k-10k%2529.osk/file

 if this can be fixed thank you

**Comment by bdach** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/12057

The long note sprite is 40k pixels high.
